doc/kernel/usermode/kernelobjects.rst

```diff
@@ -56,19 +56,10 @@ call APIs, several conditions must be met on how the kernel object is declared:
   in a special table of kernel object metadata.  Kernel objects may be members
   of arrays or embedded within other data structures.
 
-* Kernel objects must be located in memory reserved for the kernel. If
-  :option:`CONFIG_APPLICATION_MEMORY` is used, all declarations of kernel
-  objects inside application code must be prefixed with the :c:macro:`__kernel`
-  attribute so that they are placed in the right memory sections. The APIs for
-  statically declaring and initializing kernel objects (such as
-  :c:macro:`K_SEM_DEFINE()`) automatically do this. However, uninitialized
-  kernel objects need to be tagged like this:
+* Kernel objects must be located in memory reserved for the kernel. They
+  must not be located in any memory partitions that are user-accessible.
 
-.. code-block:: c
-
-    __kernel struct k_sem my_sem;
-
-* Any memory reserved for a kernel object must be used exclusively for that
+* Any memory reserved for a kernel object must be used exclusively for tha
```

Comment:
I suspect the last word should be ``that``

Reply:
fixed